### PR TITLE
Add new company - Beyond Company

### PR DESCRIPTION
**Pull request by DaviSMoura**
This Pull Request should adhere to the repository's [Code of Conduct](../../main/.github/CONTRIBUTING.md).

Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]` 

Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

Check off this list as appropriate (add an `x` between the `[]`). It will be used by @dougaitken when reviewing your contribution.

- [] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply.**
- [x] You are an employee of the company mentioned and confirm all included details are correct.
- [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
- [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: company-profiles/beyondcompany.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# BeyondPricing
```
Comment (KieranRobson) [inline company-profiles/beyondcompany.md:1]: This should be called `Beyond Company` to match the readme

File: company-profiles/beyondcompany.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+- Recife - Brazil
+- Rio de Janeiro - Brazil
+- São Paulo - Brazil
+- Belo Horizonte - Brazil
+- Brasília - Brazil
+
+## How to apply
+
+Send email to curriculo+githubremotejobs@beyondcompany.com.br
```
Comment (KieranRobson) [inline company-profiles/beyondcompany.md:38]: Is there a website for careers, if not is `curriculo+githubremotejobs@beyondcompany.com.br` correct, specifically the `.com.br`?

Comment (DaviSMoura) [reply company-profiles/beyondcompany.md:38]: No. Currently `beyondcompany.com.br` is only a landing page. Currently we accept applying only by this mailbox.

**Comment by dougaitken** [comment]
Hey @DaviSMoura - can you clarify the company name and how it is presented/stylised, please? 

The website shows `beyond co.` but you've mentioned here it's Beyond Company, and also BeyondCompany. There are build checks that will fail if the company name doesn't match the profile file name and the name in the title.

**Comment by DaviSMoura** [comment]
Hello @dougaitken . The name of company is "Beyond Company". I've just updated all files to use "Beyond Company". As you said, also you can find "beyond co." name in some places... It's a branding name. The file about Beyond Company says "Beyond Company (aka Beyond Co.)"

**Review by dougaitken — APPROVED** [review]

**Comment by dougaitken** [comment]
Hey there, thanks for contributing, it is appreciated @DaviSMoura

**Pull request merged**